release/package/run.sh

```diff
@@ -18,6 +17,7 @@ SUPPORTED_BUILD_TYPES=(
 )
 
 PROJECT_ROOT="$SCRIPT_DIR/../.."
+# TODO(gitbuda): Toolchain is now specific for a given OS -> ADJUST.
```

Comment:
NOTE: this is about, e.g., ubuntu-18.04 uses toolchain-v4, while ubuntu-22.04 uses toolchain-v5 (deprecation process for OSes)